src/data/faq.json

```diff
@@ -2209,9 +2209,7 @@
                                 "anchor": "migration_Apple",
                                 "active": false,
                                 "textblock": [
-                                    "<b>UPDATE</b><br/>This problem has been solved with version 1.5.0.<hr/> If you transfer your data to a new iPhone using iTunes, or restore a backup, the app does not work properly. On the Status tabs ('Start Screens' before version 2.12), you end up in a loop, or the exposure logging cannot be activated. The risk status is not displayed.",
-                                    "This is a known problem with backing up the random IDs, see also <a href='#backup' target='_blank' >'Will the collected random IDs be included when I backup data on my phone?'</a>.",
-                                    "Please reinstall the Corona-Warn-App on your new iPhone."
+                                    "The problem has been fixed with app version 1.5.0. You can update the Corona Warning app to the latest version [ios.current-app-version] in the <a href='https://apps.apple.com/de/app/corona-warn-app/id1512595757' target='_blank' rel='noopener noreferrer'>Apple App Store</a>."

```

Comment:
"Corona Warning app" => "Corona-Warn-App"

This is the name consistently used also in English language texts.